lib/twofishes/result.rb

```diff
@@ -36,21 +24,21 @@ def lng
       feature.geometry.center.lng
     end
 
-    def coordinates
-      [lat, lng]
+    def country_code
+      feature.cc
     end
 
-    def parents
-      @data.parents.map { |parent| Twofishes::Result.new(parent) }
+    def coordinates
+      [lat, lng]
     end
 
-    def method_missing(method_sym, *arguments, &block)
-      @data.send(method_sym)
+    def method_missing(name, *args, &block)
```

Comment:
Ah, because we camelize on the fly :smile: 
I just implemented
```
    def_delegators :feature, *GeocodeFeature::FIELDS.map { |_, v| v[:name] }
```
and then the tests failed because `display_name` wasn't there :laughing: